server.Dockerfile

```diff
@@ -3,8 +3,10 @@ MAINTAINER David Lawrence "david.lawrence@docker.com"
 
 RUN apk add --update git gcc libc-dev && rm -rf /var/cache/apk/*
 
-# Install SQL DB migration tool
-RUN go get github.com/mattes/migrate
+# Pin to the specific v1 version
+RUN git clone -b v1 https://github.com/mattes/migrate.git /go/src/github.com/mattes/migrate/
+RUN go get -u -v github.com/mattes/migrate && \
```

Comment:
Removed `-u` flag and collapsed into one layer.

I tried skipping the `go get` and ran into dependency issues on the build, so left as is:
```
Cloning into '/go/src/github.com/mattes/migrate'...
src/github.com/mattes/migrate/driver/ql/ql.go:13:2: cannot find package "github.com/cznic/ql/driver" in any of:
	/usr/local/go/src/github.com/cznic/ql/driver (from $GOROOT)
	/go/src/github.com/cznic/ql/driver (from $GOPATH)
...
```